DAY_TWO_README.md

```diff
@@ -15,26 +15,62 @@
 - The props being passed to each component should be typed checked using `propTypes` and `defaultProps`.
 - The comment section should add a comment to the post
 
+
```

Comment:
Nice job getting the authentication to work. Try following how Lambda lays it out in the readme next time.